src/com/google/javascript/jscomp/parsing/IRFactory.java

```diff
@@ -196,6 +196,10 @@
 
   static final String UNDEFINED_LABEL = "undefined label \"%s\"";
 
+  static final String EXPOSE_DEPRECATED_WARNING =
```

Comment:
Modify this file
https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/RhinoErrorReporter.java
to match the text of the warning, and create a DiagnosticGroup, so that the warning can be suppressed. Add your new diagnostic here.
https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/DiagnosticGroups.java#L127